session_linux.go

```diff
@@ -79,18 +79,16 @@ func newPaDevice(
 
 	var key string
 
-	if isOutput {
-		if isMaster {
+	// If device is the master, check if its the mic or speaker and give apporpriate default name
```

Comment:
Typo: `appropriate`

Reply:
(although this is a fairly obvious `if` statement, I'm not sure it needs commenting in the first place..)